ztp/gitops-subscriptions/argocd/example/policygentemplates/common-ranGen.yaml

```diff
@@ -72,7 +72,7 @@ spec:
         displayName: disconnected-redhat-operators
         image: registry.example.com:5000/disconnected-redhat-operators/disconnected-redhat-operator-index:v4.9
     - fileName: DisconnectedICSP.yaml
-      policyName: "registry-policy"
+      policyName: "config-policy"
```

Comment:
If the user chooses to adapt an existing deployment to this new convention I believe it will result in duplicate/parallel policies. We don't have a good way to automatically "move" already existing policies to the new names. At a minimum we need to document for the user how to clean up the old policies, but we should think about ways to make this easier if we can.

Reply:
So basically we need to help user migrate and since this feature is actually benefiting everyone without a loss in functionality it could be made default? 

To help migrate: we can introduce a special variable `migrateToMakeItBetter` (maybe added using patch), which basically deletes the old ones and recreates policies with the same name? 

To help make it default: on the docs say `policyName` is "something you shouldn't be explicit about" and then we programatically add `config-policy` to it when it's missing/empty. 

Migration might be overkill (used by small number of users and for a limited amount of time?) but default behaviour should be implanted I think. Seeing `policyName: "config-policy"` in so many places is bad UX IMO

Reply:
When the new image is instantiated, couldn't we just have a one shot cleanup that deletes the redundant policies ?

Reply:
Yes. Automating the move/cleanup/rename/etc is something we could (and should) do. I suspect that we will need (or want) traceability between source PolicyGenTemplate and created artifacts to make this easier.